src/Home.svelte

```diff
@@ -0,0 +1,242 @@
+                          $output[selectedView].verifiableCredential[field]
+                        )[
+                          Object.keys(
+                            $output[selectedView].verifiableCredential[field]
+                          ).length - 1
+                        ] === subField}
+                      >
+                        <p class="font-bold mr-2">
+                          {decamelize(subField, " ")
+                            ? `${decamelize(subField, " ")}:`
+                            : ""}
+                        </p>
+                        <p class:font-bold={!decamelize(subField, " ")}>
+                          {$output[selectedView].verifiableCredential[field][
+                            subField
+                          ]}
+                        </p>
+                      </div>
+                    {/each}
+                  </div>
+                {/if}
+              {/each}
+              <h2 class="mt-8">Proof Options</h2>
+              <!-- TODO: Refactor into component -->
+              {#each Object.keys($output[selectedView].proofOptions) as field}
+                <div class="flex flex-wrap m-2">
+                  <p class="capitalize font-bold">
+                    {decamelize(field, " ")}:&nbsp;
+                  </p>
+                  {#if typeof $output[selectedView].proofOptions[field] === "string"}
+                    <p>
+                      {$output[selectedView].proofOptions[field]}
+                    </p>
+                  {/if}
+                </div>
+                {#if typeof $output[selectedView].proofOptions[field] !== "string"}
+                  <div class="flex-col flex">
+                    {#each Object.keys($output[selectedView].proofOptions[field]) as subField}
+                      <div
+                        class="flex pl-2 ml-2 border-l-4 border-white"
+                        class:justify-center={!decamelize(subField, " ")}
+                        class:border-b-4={Object.keys(
+                          $output[selectedView].proofOptions[field]
+                        )[
+                          Object.keys($output[selectedView].proofOptions[field])
+                            .length - 1
+                        ] === subField}
+                      >
+                        <p class="font-bold mr-2">
+                          {decamelize(subField, " ")
+                            ? `${decamelize(subField, " ")}:`
+                            : ""}
+                        </p>
+                        <p class:font-bold={!decamelize(subField, " ")}>
+                          {$output[selectedView].proofOptions[field][subField]}
+                        </p>
+                      </div>
+                    {/each}
+                  </div>
+                {/if}
+              {/each}
+              {#if $output[selectedView].unknown.body.length > 0}
+                <h2 class="mt-8">Unrecognized terms</h2>
```

Comment:
Could/should this be emphasized somehow as a warning or negative condition? Ideally (I think) the explainer recognizes and explains all terms. If there are unrecognized terms, we don't know what they mean. This constitutes a semantic "hole" in processing the data. It could be problematic if the user (and this application) are expecting some kind of data but the data actually says something contrary or unexpected.

Reply:
Sure, we could make it red for start, and maybe give a little explanation about why that is there